packs/data/archetypes.db/mummy.json

```diff
@@ -0,0 +1,31 @@
+{
+    "_id": "1vxlKEYwMtPtl3hq",
+    "name": "Mummy",
```

Comment:
This journal isn't needed, we just have not removed this compendium yet. Got distracted by other things.

Reply:
Removed